.github/workflows/django.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           path: django
       - uses: actions/setup-python@v2
         with:
-          python-version: "3.8"
+          python-version: '3.8'
```

Comment:
can you revert this change or make this in another PR?

Reply:
Will do!